introduction.md

```diff
@@ -8,5 +8,5 @@ I like:
 * NGINX
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?